### ISSUE TITLE
Function to read private key from the keystore

### ISSUE THREAD

**Issue by pcppcp** [raiden-network/raiden-libs#32]
A helper to read user's ethereum keys from a keystore is already included in uRaiden, waiting to be extracted and added to `raiden-libs`.

https://github.com/raiden-network/microraiden/blob/master/microraiden/utils/private_key.py

**Comment by pcppcp** [comment]
A very annoying api indeed:

```
>>> w3.eth.account.decrypt(x.read(), '')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/xoza/.pyenv/versions/3.6.0/envs/test_360/lib/python3.6/site-packages/web3/eth.py", line 64, in account
    "This feature is disabled, pending security audit. "
AttributeError: This feature is disabled, pending security audit. If you want to use unaudited code dealing with private keys, despite the risks, you can run `w3.eth.enable_unaudited_features()` and try again.
```

We can use web3, sure.
But still `get_private_key()` is _very_ useful. It can use file or readline key input, and raw, hex-encoded keyfiles.

**Comment by palango** [comment]
Yeah, it's not ready for primetime and it might be worth to have a wrapper. But in the long term we should minimize our own code and try to upstream as much as possible.

**Comment by palango** [comment]
Fixes by #33
